x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/tag_display.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiPopover, EuiBadgeGroup, EuiBadge, EuiButtonEmpty } from '@elastic/eui';
+import styled from 'styled-components';
+import * as i18n from '../translations';
+
+interface TagsDisplayProps {
+  tags: string[];
+}
+
+const TagWrapper = styled(EuiBadgeGroup)`
+  width: 100%;
+`;
+
+const TagPopoverWrapper = styled(EuiBadgeGroup)`
+  max-height: 200px;
+  max-width: 600px;
+  overflow: auto;
+`;
+
+const TagPopoverButton = styled(EuiButtonEmpty)`
+  font-size: ${({ theme }) => theme.eui.euiFontSizeXS}
+  font-weight: 500;
+  height: 20px;
+`;
+
+/**
+ * @param tags to display for filtering
+ */
+const TagsDisplayComponent = ({ tags }: TagsDisplayProps) => {
```

Comment:
nit: Similar to the `TagsFilterPopover`,  it might be a good idea to sort the tags for consistency to enable users to easily scan through tags.

https://github.com/elastic/kibana/blob/bb163a919d30362b8091770ef24c5dc76bf11bc6/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_table_filters/tags_filter_popover.tsx#L65-L67